guides/micronaut-data-jdbc-repository/micronaut-data-jdbc-repository.adoc

```diff
@@ -1,47 +1,63 @@
 include::{commondir}/common-header.adoc[]
 
-The application exposes some REST endpoints and stores data in a database using Micronaut Data JDBC.
+The application exposes some REST endpoints and stores data in a MySQL database using Micronaut Data JDBC.
 
-include::{commondir}/common-requirements.adoc[]
+== What you will need
+
+To complete this guide, you will need the following:
+
+* Some time on your hands
+* A decent text editor or IDE
+* JDK 1.8 or greater installed with `JAVA_HOME` configured appropriately
+* https://www.docker.io/gettingstarted/#h_installation[Docker] installed to run MySQL and for running tests, using https://www.testcontainers.org[Testcontainers].
 
 include::{commondir}/common-completesolution.adoc[]
 
-include::{commondir}/common-create-app.adoc[]
+== Writing the Application
 
-=== Data Source configuration
+include::{commondir}/common-cli-or-launch.adoc[]
 
-Add the following dependencies:
+:exclude-for-languages:groovy
```

Comment:
we should support: 

`mn create-app --features=@features example.micronaut.micronautguide --build=@build@ --lang=@lang@`

We should be generate the correct list of features for java, kotlin and groovy (removing graalvm)